tests/conftest.py

```diff
@@ -340,3 +127,231 @@ def memory_jobstore():
     store.connect()
 
     return store
+
+@job
+def mock_rss(input_dir: str = None,
```

Comment:
I am not sure if I understand the point of this. Do we really need this? Shouldn't we rather actually test https://github.com/QuantumChemist/autoplex/blob/7884e636b6f0feb995cde990a93c8732dcee412f/src/autoplex/auto/rss/flows.py#L14  defined that actually do these steps and have that tested instead?

Reply:
I opened this issue https://github.com/autoatml/autoplex/issues/293